src/xvelopers.json

```diff
@@ -1,7 +1,7 @@
 [
   {
-    "id":"luix",
-    "name": "Luix Ramirez",
+    "id":"lui",
+    "name": "Elian Luix Ramirez",
```

Comment:
Hola,

Esos fueron cambios hechos en la clase de Videojuegos en Intec. Favor
omitir el pull request.
El mar. 22, 2016 9:30 PM, "Enmanuel Toribio" <notifications@github.com>
escribió:

> In src/xvelopers.json
> <https://github.com/eatskolnikov/xvelopers/pull/21#discussion_r57097664>:
>
> > @@ -1,7 +1,7 @@
> >  [
> >    {
> > -    "id":"luix",
> > -    "name": "Luix Ramirez",
> > +    "id":"lui",
> > +    "name": "Elian Luix Ramirez",
>
> Jajjajajajjaja. ¿Quien es Elian Luix Ramirez? @-@
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/eatskolnikov/xvelopers/pull/21/files/6dc9d2f7c1ad214bf1dfbe2ef43a4eb202b2da1a#r57097664>
>